resources/audioPlayer.go

```diff
@@ -0,0 +1,102 @@
+package resources
+
+import (
+	"github.com/faiface/beep"
+	"github.com/faiface/beep/mp3"
+	"github.com/faiface/beep/speaker"
+	"log"
+	"os"
+	"time"
+)
+
+type sound struct {
+	path              string
+	isBackgroundMusic bool
+}
+
+var sounds = map[string]sound {
```

Comment:
is there actually a reason we need a map? using global constants for each one would be faster and i think would make more sense than referring to strings

Reply:
guess it doesnt matter tho since the strings are only referred to in 1 place